packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -376,6 +382,10 @@ export interface GridOptions {
    * Callback fired when the state of the grid is updated.
    */
   onStateChange?: (params: any) => void;
+  /**
+   * Callback fired when the grid is unmounted.
+   */
+  onUnmount?: (params: any) => void;
```

Comment:
Are we sure about this one? That's the use case? At first sight, it goes against the hook philosophy. And developers can implement this behavior on their side, they control the mounting state of the grid.

Reply:
I agree. It can be done independently of the option. It's common practice to handle that case yourself @dtassone

Reply:
Removed